classes/cluster/mk22_full_scale/fuel/config.yml

```diff
@@ -43,33 +43,25 @@ parameters:
       environment:
         dev:
           formula:
-            cinder:
-              revision: stacklight
-            collectd:
-              revision: stacklight
-            galera:
-              revision: stacklight
-            glance:
-              revision: stacklight
-            haproxy:
-              revision: stacklight
-            heat:
-              revision: stacklight
+            grafana:
+              state:
+                grafana3_datasource.py:
+                  enabled: true
+                grafana3_dashboard.py:
+                  enabled: true
```

Comment:
It looks like this is not necessary anymore, no? See https://github.com/Mirantis/mk-lab-salt-model/pull/144/files.